cookbook: Improve the introduction page (#18)
rust-bitcoin/www.rust-bitcoin.org: cookbook/src/intro.md

# Welcome!

This book is created and maintained by those involved in the
[`rust-bitcoin`](https://github.com/rust-bitcoin) GitHub organization, contributions are
appreciated. It covers various crates from the org and as such, aims to be useful to developers
wanting to write code in Rust that interacts with the Bitcoin network. It is specifically not
limited to just the [`rust-bitcoin`](https://github.com/rust-bitcoin/rust-bitcoin) crate, although

> that is a good starting point if you want a one stop shop for interacting with Bitcoin in Rust.

Reviewer: ```suggestion
that is a good starting point if you want a one-stop-shop for interacting with Bitcoin in Rust.
```

> There are a number of good libraries built outside of the `rust-bitcoin` organization that build on
> top of the crates covered here, two that you might like to check out are:

Reviewer: ```suggestion
There are a number of good libraries outside of `rust-bitcoin` organization that uses the crates covered here, two that you might like to check out are:
```

Author: Used:
```
There are a number of good libraries outside of the `rust-bitcoin` organization that use the crates
covered here, two that you might like to check out are:
```

- [`Bitcoin Dev Kit`](https://bitcoindevkit.org/)
- [`Lightning Dev Kit`](https://lightningdevkit.org/)

> Finally, this book is currently WIP but hopes to eventually cover various topics, including parsing

Reviewer: ```suggestion
Finally, this book is currently Work in Progress (WIP) but hopes to eventually cover various topics, including parsing
```

Don't assume people know all TLAs

Author: Used:
```
Finally, this book is currently a work inn progress but hopes to eventually cover various topics,
including parsing blocks and transactions, constructing and signing transactions, receiving data
over the peer-to-peer network, plus fun stuff you can do with miniscript.
```

> blocks and transactions, constructing and signing transactions, receiving data over P2P, plus fun

Reviewer: ```suggestion
blocks and transactions, constructing and signing transactions, receiving data over Peer-to-Peer (P2P), plus fun
```

TLAs

Author: thanks, covered above.
